edi_af_bar_arbetsuppgifter/views/edi_af_as_notes_views.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <odoo>
-    <record id="af_edi_as_notes" model="ir.actions.server">
-        <field name="name">EDI AF post note</field>
+    <record id="af_edi_bar" model="ir.actions.server">
+        <field name="name">EDI AF bar</field>
```

Comment:
Itt's called "bär" in edi_route_data.xml and "bar" here. 
should it be "bär" here too?